static/xml/ns/jakartaee/web-common_6_0.xsd

```diff
@@ -75,9 +76,9 @@
     </xsd:documentation>
   </xsd:annotation>
 
-  <xsd:include schemaLocation="jakartaee_10.xsd"/>
+  <xsd:include schemaLocation="jakartaee_9.xsd"/>
```

Comment:
Probably not correct.
File generated by me now has `jakartaee_10` included here.